Code/MinimalLib/jswrapper.cpp

```diff
@@ -553,14 +569,26 @@ EMSCRIPTEN_BINDINGS(RDKit_minimal) {
       .function("straighten_depiction",
                 select_overload<void()>(&JSMol::straighten_depiction))
       .function("straighten_depiction",
-                select_overload<void(bool)>(&JSMol::straighten_depiction));
+                select_overload<void(bool)>(&JSMol::straighten_depiction))
+      .function("get_num_atoms", select_overload<unsigned int(bool) const>(
+                                     &JSMol::get_num_atoms))
+      .function("get_num_atoms",
+                select_overload<unsigned int() const>(&JSMol::get_num_atoms))
+      .function("get_num_bonds", &JSMol::get_num_bonds)
+  ;
 
   class_<JSMolIterator>("MolIterator")
+      .constructor<>()
```

Comment:
Once you add write access to this, having it be called `MolIterator` is quite misleading (at least in terms of naming in the rest of the RDKit, where things like atom or bond iterators operate over an existing sequence but do not allow the sequence itself to be modified).

Rather than repurposing this one class to be both a non-modifiable sequence of objects (like what get_mol_frags() returns), I'd prefer to have a new class, perhaps `MolList` for this. 

Alternatively you could just rename this particular class, but I suspect that may break existing code? Maybe not since there would be no reason for people to be constructing `MolIterator` objects themselves from JS at the moment.

Reply:
I agree, I had the same thoughts honestly.
I will just rename this class to `MolList`.
Previously it was not possible to construct a `MolIterator` from JS - it was only returned as object by `JSMol::get_frags()`, so this should not be problem.
The only developer who is using the `MolIterator` constuctor should be me as I am already using the not-yet-reviewed code in this branch.
So I think I can convince myself to rename `MolIterator` to `MolList` in my own code - I will use good manners and diplomacy.

Reply:
There is actually a place where this would break existing code: the object returned by `JSMol::get_frags()` as a dictionary with two keys, currently `molIterator` and `mappings`. Again, I prefer to introduce a breaking change than to stick to an inappropriate key name. And most likely there are very few users of the `get_frags` API (possibly just me). I will add a note to Release Notes about the breaking change and go ahead.

Reply:
Done, and added the `2023_09_1` milestone.